cmd/integration/integration.go

```diff
@@ -82,7 +82,7 @@ func (fakeKubeletClient) GetPodStatus(host, podNamespace, podID string) (api.Pod
 	default:
 		glog.Fatalf("Can't get info for: '%v', '%v - %v'", host, podNamespace, podID)
 	}
-	r, err := c.GetPodStatus("localhost", podNamespace, podID)
+	r, err := c.GetPodStatus("127.0.0.1", podNamespace, podID)
```

Comment:
This is more correct - we're saying "use the host at 127.0.0.1" instead of "resolve localhost".